src/shared/components/VideoPlayer/videoJsPlayer.ts

```diff
@@ -1,5 +1,6 @@
 import { RefObject, useEffect, useRef, useState } from 'react'
-import videojs, { VideoJsPlayer, VideoJsPlayerOptions } from 'video.js'
+import { VideoJsPlayer, VideoJsPlayerOptions } from 'video.js'
+import videojs from 'video.js/dist/alt/video.core.novtt'
```

Comment:
Just out of curiosity - this will reduce bundle size?

Reply:
Yes, this stripped around 700kb (uncompressed) from the bundle